src/components/BaseCard.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="flex flex-col rounded-lg shadow-lg overflow-hidden">
+    <div class="flex-shrink-0">
+      <img class="h-48 w-full object-cover" :src="src">
+    </div>
+    <div class="flex-1 bg-white p-6 flex flex-col justify-between">
+      <div class="flex-1">
+        <p class="text-sm font-medium text-indigo-600">
```

Comment:
There is no need to have the anchor tag inside the p tag, you can directly apply the classes on the anchor tag and remove the p tag